### PR TITLE
feat: Sole Sans font

### PR DESCRIPTION
**Pull request by arnest00**
### Description
- Add Sole Sans to sans font stack and apply to `:root` element
- Add link to `assets.sparkbox.com` font stylesheet to `_document.jsx`
- Add remaining SCSS partials to `globals.scss`

### Spec
See Story: [205](https://sparkbox.atlassian.net/browse/FSA22V1-205)

### Validation
* [ ] This PR has code changes, and our linters still pass.
* [ ] This PR affects production code, so it was browser tested.

#### To Validate
1. Pull down `feat--sole-sans-font` branch.
2. Run `npm install`.
3. Run `npm run lint` and verify all linters pass.
4. Run `npm run test` and verify all tests pass.
5. Navigate to deploy preview and verify body text is rendered in Sole Sans font.

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": ["stylelint-config-standard", "stylelint-config-standard-scss"], 
   "rules": {
-    "value-list-comma-newline-after": "always-multi-line"
+    "value-list-comma-newline-after": "always-multi-line",
+    "function-no-unknown": null
```
[the remark this answers is not in the document]
Comment (dustin-jw) [reply .stylelintrc.json:5]: Disabling the rule works, but I think it's probably better to use the `ignoreFunctions` property to specify our custom functions. That way it'll catch typos with standard CSS functions as well as our custom functions. https://stylelint.io/user-guide/rules/list/function-no-unknown/

File: .stylelintrc.json [hunk 2]
```diff
@@ -1,6 +1,16 @@
 {
   "extends": ["stylelint-config-standard", "stylelint-config-standard-scss"], 
   "rules": {
-    "value-list-comma-newline-after": "always-multi-line"
+    "value-list-comma-newline-after": "always-multi-line",
+    "function-no-unknown": [
+      true,
+      { "ignoreFunctions": [
```
Comment (dustin-jw) [inline .stylelintrc.json:7]: Just the tiniest white-space nitpick: can we move `ignoreFunctions` down to the next line so the curly brace is all by itself?

**Review by dustin-jw — DISMISSED** [review]
I just have one tiny suggestion that I won't let block the PR, so I'll pass this over to @howema to finish reviewing!

**Review by theogainey — APPROVED** [review]
Looks good!

**Pull request merged**